setup.py

```diff
@@ -53,7 +53,8 @@
     include_package_data=True,
     install_requires=[
         "eth-hash>=0.3.1",
-        "eth-typing>=3.0.0",
+        "eth-typing>=5.0.0b1",
+        "hexbytes>=1.0.0",
```

Comment:
@reedsa  I do not see this actually being used and it currently creates a cyclic dependency: https://github.com/ethereum/hexbytes/blob/main/setup.py#L24